cukedoctor-multipage-layout/src/main/java/cukedoctor/multipage/api/MultipageConverter.java

```diff
@@ -0,0 +1,74 @@
+package cukedoctor.multipage.api;
+
+import com.github.cukedoctor.Cukedoctor;
+import com.github.cukedoctor.api.CukedoctorConverter;
+import com.github.cukedoctor.util.FileUtil;
+import com.github.cukedoctor.util.ServiceLoaderUtil;
+import cukedoctor.multipage.defaults.DefaultMultipagePersister;
+import cukedoctor.multipage.defaults.OnePagePerFeatureMultipagePager;
+import cukedoctor.multipage.model.Page;
+import cukedoctor.multipage.model.MultipageDocumentation;
+import cukedoctor.multipage.spi.MultipagePersister;
+import cukedoctor.multipage.spi.MultipagePager;
+
+import java.util.List;
+
+/**
+ * This class serves as a replacement to the {@link com.github.cukedoctor.converter.CukedoctorConverterImpl} and can
+ * handle multipage documentation.
+ *
+ * Main usage is to create a documentation sites with tools like antora.
+ */
+public class MultipageConverter {
+    private MultipagePager pager;
+    private MultipagePersister persister;
+
+    private String jsonFilesLocation = "target/test-classes/json-output/";
+    private String outputFolderLocation = "docs/modules/living/pages/";
+
+    public MultipageConverter() {
+        loadDependencies();
+    }
+
+    private void loadDependencies() {
+        this.pager = new ServiceLoaderUtil<MultipagePager>().initialise(MultipagePager.class, OnePagePerFeatureMultipagePager.class, null, null, null);
+        this.persister = new ServiceLoaderUtil<MultipagePersister>().initialise(MultipagePersister.class, DefaultMultipagePersister.class, null, null, null);
+    }
+
+    private MultipageDocumentation generateMultipageDocumentation() {
+        List<String> pathToCucumberJsonFiles = FileUtil.findJsonFiles(jsonFilesLocation);
+        List<Page> pages = pager.pages(pathToCucumberJsonFiles);
+
+        MultipageDocumentation multipageDocumentation = new MultipageDocumentation(pages);
+
+        for (Page page : pages) {
+            CukedoctorConverter converter = Cukedoctor.instance(page.getFeatures(), page.getDocumentAttributes());
+            converter.setFilename("docs/modules/living/pages/" + page.getPageTitle() + ".adoc");
+            page.setRender(converter.renderDocumentation());
+        }
```

Comment:
Here we need to remove summary and may `step time` and `feature/scenario keyword`:

```suggestion
MultipageDocumentation multipageDocumentation = new MultipageDocumentation(pages);
CukedoctorConfig config = new CukedoctorConfig().setHideSummarySection(true)
                .setHideFeaturesSection(true)
                .setHideStepTime(true)
                .setHideScenarioKeyword(true)
                .setHideTags(true);
        for (Page page : pages) {
            CukedoctorConverter converter = Cukedoctor.instance(page.getFeatures(), page.getDocumentAttributes(), config);
            converter.setFilename("docs/modules/living/pages/" + page.getPageTitle() + ".adoc");
            page.setRender(converter.renderDocumentation());
        }
```